lib/oli_web/live/delivery/manage_updates.ex

```diff
@@ -9,8 +9,24 @@ defmodule OliWeb.Delivery.ManageUpdates do
   alias Oli.Delivery.Updates.Worker
   alias OliWeb.Delivery.Updates.ApplyUpdateModal
   alias Oli.Delivery.Updates.Subscriber
+  alias OliWeb.Common.Breadcrumb
   alias OliWeb.Sections.Mount
 
+  def set_breadcrumbs(section, type) do
+    OliWeb.Sections.OverviewView.set_breadcrumbs(type, section)
```

Comment:
If we're going to use the pipe, we should use it all the way - so start with type |> OliWeb....